scripts/install

```diff
@@ -162,27 +161,26 @@ done
 #=================================================
 # SECURE FILES AND DIRECTORIES
 #=================================================
+ynh_script_progression --message="Securing files and directories..."
 
 # Set permissions to app files
-chown -R root: $final_path
-chown pepettes:root $final_path
-chown pepettes:root $final_path/settings.py
-chmod o=--- $final_path/settings.py
+chown -R $app:www-data $final_path
+chmod o=--- $final_path
 
 #=================================================
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-yunohost service add $app --description="A simple donation form" --log="/var/log/$app/$app.log"
+yunohost service add $app --description="A simple donation form" --log_type="systemd"
 
 #=================================================
 # START SYSTEMD SERVICE
 #=================================================
 ynh_script_progression --message="Starting a systemd service..." --weight=1
 
 # Start a systemd service
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" --line_match="Started"
```

Comment:
Not sure to understand why that doesn't work, but on ci-apps-dev : 

```
333744 INFO [###############+....] > Starting a systemd service...
635556 WARNING Please wait, the service pepettes is starting..........................................................................................................................................................................................................................................................................................................
635558 WARNING The service pepettes didn't fully executed the action start before the timeout.
635558 WARNING Please find here an extract of the end of the log of the service pepettes:
635564 WARNING Mar 10 23:35:53 systemd[1]: Started pepettes Daemon.
```

i.e. it doesn't find the "Started" despite the fact that it's in the systemd journal :/

Reply:
... It's something I've already seen and that we discuss at:
- https://github.com/YunoHost/yunohost/pull/865
- https://github.com/YunoHost/yunohost/pull/959
- https://github.com/YunoHost/issues/issues/1573

Related to ynh_systemd_action not catching the first lines of log when the service start too quickly ...